plugins/lookup/tss.py

```diff
@@ -36,15 +36,15 @@
         ini:
             - section: tss_lookup
               key: username
-        required: true
+        required: false
     password:
         description: The password associated with the supplied username.
```

Comment:
I would mentioned when this is needed (same above):
```suggestion
        description:
            - The password associated with the supplied username.
            - Required when I(token) is not provided.
```